src/main/resources/assets/wurst/lang/ko_KR.json

```diff
@@ -0,0 +1,371 @@
+  "description.wurst.altmanager.premium": "This alt has a password and can join all servers.",
+  "description.wurst.altmanager.cracked": "This alt has no password and will only work on cracked servers.",
+  "description.wurst.altmanager.failed": "Last time you tried to log in with this alt, it didn't work.",
+  "description.wurst.altmanager.checked": "The password has worked in the past.",
+  "description.wurst.altmanager.unchecked": "You have never successfully logged in with this alt.",
+  "description.wurst.altmanager.favorite": "You have marked this alt as one of your favorites.",
+  "description.wurst.altmanager.window": "This button opens another window.",
+  "description.wurst.altmanager.window_freeze": "It might look like the game is not responding while that window is open.",
+  "description.wurst.altmanager.fullscreen": "§cTurn off fullscreen mode!",
+  "gui.wurst.altmanager.folder_error.title": "Couldn't create the '.Wurst encryption' folder!",
+  "gui.wurst.altmanager.folder_error.message": "You may have accidentally blocked Wurst from accessing this folder.\nAltManager cannot encrypt or decrypt your alt list without it.\nYou can still use AltManager, but any alts you create now won't be saved.\n\nThe full error is as follows:\n%s",
+  "gui.wurst.altmanager.empty.title": "Your alt list is empty.",
+  "gui.wurst.altmanager.empty.message": "Would you like some random alts to get started?",
+  "gui.wurst.nochatreports.unsafe_server.title": "§4§lWARNING:§r Unsafe Server",
+  "gui.wurst.nochatreports.unsafe_server.message": "This server requires chat signatures to be enabled, which puts your account at risk of fraudulent chat reports.\n\nYou can chat on this server if you un-block chat signatures and then reconnect. If you do, consider playing on an alt account that you don't mind losing.\n\nIf this is your server, you can fix this by setting 'enforce-secure-profile' to false in server.properties. In typical Mojang fashion, this setting does the opposite of what it sounds like.",
+  "toast.wurst.nochatreports.unsafe_server.title": "Chat messages can be reported",
+  "toast.wurst.nochatreports.unsafe_server.message": "Messages sent on this server may be reported to Big Brother Mojang and might get your account banned globally",
+  "gui.wurst.nochatreports.ncr_mod_server.title": "\"No Chat Reports\" Server",
+  "gui.wurst.nochatreports.ncr_mod_server.message": "This server uses the \"No Chat Reports\" mod.\n\nYou can join this server if you block chat signatures and disable VanillaSpoof.",
+  "button.wurst.nochatreports.signatures_status": "Signatures: ",
+  "description.wurst.nochatreports.message_is_reportable": "This message has a valid signature and is thus vulnerable to fraudulent chat reports.",
+  "gui.wurst.generic.allcaps_blocked": "BLOCKED",
+  "gui.wurst.generic.allcaps_allowed": "ALLOWED"
```

Comment:
Hi @JAXPLE,

Thanks for your pull request. I appreciate your effort in translating everything and making Wurst more accessible to Korean-speaking users.

However, it seems that there are a lot of duplicated and untranslated strings at the bottom of the `ko_KR.json` file. Could you please update the file to remove any duplicates and untranslated lines?

Feel free to let me know if you encounter any difficulties or have any questions.